InteractiveAutomationToolkit/InteractiveController.cs

```diff
@@ -155,6 +155,20 @@ public void Update()
 			CurrentDialog.Show(false);
 		}
 
+		/// <summary>
+		/// Hides current dialog.
+		/// </summary>
+		/// <exception cref="InvalidOperationException">When no dialog has been set.</exception>
+		public void Hide()
```

Comment:
I wouldn't use CurrentDialog.Hide() here. But instead use Engine.HideUi(). So no need to check whether the CurrentDialog exists. According to the RN it is possible to hide the UI even if nothing was shown before.